cards/src/main/resources/cards/custom/group3/bug/kobolds_and_catacombs/minion_dea.json

```diff
@@ -6,17 +6,42 @@
   "baseAttack": 5,
   "baseHp": 5,
   "rarity": "LEGENDARY",
-  "description": "Opener: Set all the numbers in the next card you play to 1.",
+  "description": "Opener: Add a copy of a minion to your hand. Set its stats and Cost to 0.",
   "battlecry": {
-    "targetSelection": "NONE",
+    "targetSelection": "MINIONS",
     "spell": {
-      "class": "NullSpell"
+      "class": "CopyCardSpell",
+      "spell": {
+        "class": "MetaSpell",
+        "target": "OUTPUT",
+        "spells": [
+          {
+            "class": "CardCostModifierSpell",
+            "cardCostModifier": {
+              "class": "CardCostModifier",
+              "target": "SELF",
+              "value": 0,
+              "operation": "SET"
+            }
+          },
+          {
+            "class": "SetAttackSpell",
+            "value": 0,
+            "exclusive": true
+          },
+          {
+            "class": "SetHpSpell",
```

Comment:
As far as I remember, summoning a minion with zero health can cause infinite loop issues with cards that trigger off deaths and resummon the minion. There was a trace for this historically. Or maybe it was something like, whenever a minion dies, summon a Skeleton with the same stats or something.